Tasks/VsTest/taskinputparser.ts

```diff
@@ -186,7 +187,12 @@ function initTestConfigurations(testConfiguration: models.TestConfigurations) {
 
     testConfiguration.taskInstanceIdentifier = uuid.v1();
 
-    versionFinder.getVsTestRunnerDetails(testConfiguration);
+    try {
+        versionFinder.getVsTestRunnerDetails(testConfiguration);
+    } catch (error) {
+        utils.Helper.publishEventToCi(AreaCodes.SPECIFIEDVSVERSIONNOTFOUND, error.message, 1039, true);
+        throw error;
```

Comment:
make sure while testing that this is propagated to the final catch

Reply:
validated